nb2/05.02-V2-Introducing-Scikit-Learn.ipynb

```diff
@@ -2,10 +2,7 @@
  "cells": [
```

Comment:
This is comment 0.
<br/>

 _Reply via <a href='http://127.0.0.1:8000/amit1rrr/PythonDataScienceHandbook/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>